snomed/com.b2international.snowowl.snomed.datastore.server/src/com/b2international/snowowl/datastore/server/snomed/version/SnomedPublishManager.java

```diff
@@ -80,9 +77,9 @@
 public class SnomedPublishManager extends PublishManager {
 
 	private Set<String> componentIdsToPublish = newHashSet();
-	
-	// moduleId to referencedComponentId
-	private Collection<Pair<String, String>> moduleToReferenceComponentIds;
+
+	// sourceModuleId to targetModuleId map
+	private Map<String, String> moduleDependencies;
```

Comment:
This should be a `Multimap` so that multiple targets can be added for a single source module.